src/poetry/packages/locker.py

```diff
@@ -397,6 +398,7 @@ def set_lock_data(self, root: Package, packages: list[Package]) -> bool:
             del package["files"]
 
         lock = document()
+        lock.add(comment("@" + "generated"))
```

Comment:
Let's make this a longer form comment -- something like:

```toml
# This file is automatically @generated by Poetry and should not be changed by hand.
```

Reply:
Also, let's store that string in a constant, then import it in the tests instead of duplicating it.

Reply:
Yeah, I think I'm going to ask that you please pull in the string from the constant instead of repeating it -- both because of the need to not have a literal `@generated` as well as in the name of DRY I think it's better. Other tests do include hardcoded values, but in some cases that's the only way, and in others it just needs refactoring.